### PR TITLE
Basic skeleton and feature

### PR DESCRIPTION
**Pull request by CrymSonir**
- Create theme skeleton

- Add basic feature "Custom SCSS", "Theme Preset", "Brandcolor"

- Add a template for bug report

### CODE REVIEW

File: lang/en/theme_telaformation.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+// Every file should have GPL and copyright in the header - we skip it in tutorials but you should not skip it for real.
+// This line protects the file from being accessed by a URL directly.
+// This is the EN Lang package
+defined('MOODLE_INTERNAL') || die();
+
+// A description shown in the admin theme selector.
+$string['choosereadme'] = 'Theme teleformation is a child theme of Boost. It adds some new features';
```
Comment (kstefanini) [inline lang/en/theme_telaformation.php:8]: It's Tel**A**formation :)

File: lang/en/theme_telaformation.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+// The name of our plugin.
+$string['pluginname'] = 'Teleformation';
```
Comment (kstefanini) [inline lang/en/theme_telaformation.php:10]: Same, with an A

File: lang/fr/theme_telaformation.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?php
+// Every file should have GPL and copyright in the header - we skip it in tutorials but you should not skip it for real.
+// This line protects the file from being accessed by a URL directly.
+// This is the FR Lang package
+defined('MOODLE_INTERNAL') || die();
+
+// A description shown in the admin theme selector.
+$string['choosereadme'] = 'Le thème de la téléformation est un thème enfant de Boost. Il ajoute quelques nouvelles fonctionnalités';
```
Comment (kstefanini) [inline lang/fr/theme_telaformation.php:8]: Same, with an A

File: lang/fr/theme_telaformation.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+// The name of our plugin.
+$string['pluginname'] = 'Teleformation';
```
Comment (kstefanini) [inline lang/fr/theme_telaformation.php:10]: Same, with an A

**Comment by CrymSonir** [comment]
@kstefanini Oui j'avais vu ça arrivera avec la prochaine version 👍

**Pull request merged**